src/System.ValueTuple/ref/project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "frameworks": {
+    "netstandard1.0": {
+      "dependencies": {
+        "System.Collections": "4.3.0",
+        "System.Diagnostics.Debug": "4.3.0",
+        "System.Resources.ResourceManager": "4.3.0",
+        "System.Runtime": "4.3.0"
+      }
+    },
+    "portable-net40+sl4+win8+wp8": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Portable.v4.0.Profile36": "1.0.0"
+      }
+    },
+    "netcoreapp1.1": {
+      "dependencies": {
```

Comment:
Go ahead and add System.Runtime here as well because it will eventually need it once we publish a package with your new APIs.